infra/gcb/build_project.py

```diff
@@ -355,7 +355,10 @@ def run_build(build_steps, tag):
       'timeout': str(BUILD_TIMEOUT) + 's',
       'options': options,
       'logsBucket': GCB_LOGS_BUCKET,
-      'tags': [ tag ],
```

Comment:
Does this need to go in tags field. In original patch, i see it in 

'images': [ image ],